### ISSUE TITLE
nimble_publisher still tries to call Makeup even if no highlighters are supplied.

### ISSUE THREAD

**Issue by joseph-lozano** [dashbitco/nimble_publisher#6]
A common use case would be to want to use other syntax highlighters for code (such as highlight.js).

Nimble Publisher should be able to recognize that no highlighters were set, and not attempt to run Makeup at all. The current behavior, with no highlighters specified in the `use` call, is to error with this stacktrace

```
** (BadMapError) expected a map, got: nil
    (stdlib 3.13.2) :maps.find("elixir", nil)
    (makeup 1.0.5) lib/makeup/registry.ex:124: Makeup.Registry.fetch_lexer_by_name/1
    (nimble_publisher 0.1.1) lib/nimble_publisher/highlighter.ex:25: NimblePublisher.Highlighter.pick_language_and_lexer/1
    (nimble_publisher 0.1.1) lib/nimble_publisher/highlighter.ex:16: NimblePublisher.Highlighter.highlight_code_block/3
    (elixir 1.11.1) lib/regex.ex:731: Regex.apply_list/5
    (elixir 1.11.1) lib/regex.ex:668: Regex.do_replace/4
    (nimble_publisher 0.1.1) lib/nimble_publisher.ex:43: anonymous fn/3 in NimblePublisher.__extract__/2
    (elixir 1.11.1) lib/enum.ex:2181: Enum."-reduce/3-lists^foldl/2-0-"/3
    (nimble_publisher 0.1.1) lib/nimble_publisher.ex:41: NimblePublisher.__extract__/2
```

Happy to do the work myself to correct this, but I am wondering if this was considered at all.

**Comment by josevalim** [comment]
@joseph-lozano a PR would be welcome! :hear:

**Comment by joseph-lozano** [comment]
Closed with #7
